README.md

```diff
@@ -19,13 +19,18 @@ Easily configurable local LEMP-environment for multiple hostnames.
 * Install [Vagrant](https://www.vagrantup.com/) & [VirtualBox](https://www.virtualbox.org/wiki/VirtualBox)
 * Install [Ansible](http://docs.ansible.com/ansible/intro_installation.html)
 * Install hostsupdater-plugin for vagrant: `vagrant plugin install vagrant-hostsupdater`
+* Make sure your `ssh-agent` is running and has your SSH keys added to it
+  * Autostart `ssh-agent` on shell startup: https://help.github.com/articles/working-with-ssh-key-passphrases/#auto-launching-ssh-agent-on-git-for-windows
 * Run `vagrant up`
 
 ### Adding sites (projects)
 * Add site-information to `config.yml` `sites`-array.
 * Run `vagrant provision` to automatically add necessary information to the box & to you hosts-file.
 
 ### What if I run into problems...?
+* Sometimes the box has to be provisioned twice, since installing `yarn` occasionally throws an error
+* Insalling Mailcatcher has been disabled in `/provision/roles/tools/tasks/main.yml` for the time being, since it keeps failing
```

Comment:
I would install it with `sudo npm install -g yarn` instead, that should at least not throw any odd errors and it won't attempt to place things in your home folder.